component styles/Avatar.css

```diff
@@ -0,0 +1,52 @@
+.avatar-container{
+    display: flex;
+    width: 50%;
+    height: max-content;
+    justify-content: flex-start;
+    align-items: flex-start;
+    flex-direction: column;
+    padding: 1rem;
+}
+
+.avatar-container h1,h3{
+    margin-top: 1rem;
+    font-family: var(--primary-font);
```

Comment:
instead of doing like this better approach would be to do

```
h1, h2, h3, h4, h5, h6 {
  font-family: var(--primary-font)
}

p {
font-family: var(--secondary-font)
}
```

this allows you to write less CSS.

Reply:
Okay, Also I will move it to parent file as its common for all component. Thanks.